templates/ai/consulting.html

```diff
@@ -0,0 +1,294 @@
+            </div>
+          </div>
+        </li>
+        <li class="p-stepped-list__item">
+          <div class="row">
+            <div class="col-3">
+              <p class="p-stepped-list__title p-heading--5">Assess</p>
+            </div>
+            <div class="col-6">
+              <div class="p-stepped-list__content">Once you identify the use cases, select the key success metrics for your project and pinpoint potential blockers.</div>
+            </div>
+          </div>
+        </li>
+        <li class="p-stepped-list__item">
+          <div class="row">
+            <div class="col-3">
+              <p class="p-stepped-list__title p-heading--5">Prepare</p>
+            </div>
+            <div class="col-6">
+              <div class="p-stepped-list__content">Gather all the data and prepare it to start experimenting. Ingest, clean and extract the relevant features.</div>
+            </div>
+          </div>
+        </li>
+        <li class="p-stepped-list__item">
+          <div class="row">
+            <div class="col-3">
+              <p class="p-stepped-list__title p-heading--5">Build</p>
+            </div>
+            <div class="col-6">
+              <div class="p-stepped-list__content">Build an environment where it's easy to experiment and scale up down the road. Choose the most suitable architecture for your enterprise. For instance, public, private or hybrid, multi-cloud.</div>
+            </div>
+          </div>
+        </li>
+        <li class="p-stepped-list__item">
+          <div class="row">
+            <div class="col-3">
+              <p class="p-stepped-list__title p-heading--5">Experiment</p>
+            </div>
+            <div class="col-6">
+              <div class="p-stepped-list__content">Start experimenting and find the model that performs best. Automate workflows as you go to simplify your work.</div>
+            </div>
+          </div>
+        </li>
+        <li class="p-stepped-list__item">
+          <div class="row">
+            <div class="col-3">
+              <p class="p-stepped-list__title p-heading--5">Deploy</p>
+            </div>
+            <div class="col-6">
+              <div class="p-stepped-list__content">Deploy your model to production and start monitoring both your infrastructure and model.</div>
+            </div>
+          </div>
+        </li>
+        <li class="p-stepped-list__item">
+          <div class="row">
+            <div class="col-3">
+              <p class="p-stepped-list__title p-heading--5">Improve</p>
+            </div>
+            <div class="col-6">
+              <div class="p-stepped-list__content">Enhance model performance continuously and adapt it based on new datasets.</div>
+            </div>
+          </div>
+        </li>
+      </ol>
+    </div>
+    <hr>
+  </div>
+</section>
+
+
+<section class="p-strip">
```

Comment:
as for as the design is concerned, looks like this section has to be with the white background

Reply:
White is a way to highlight a main cta section - the preceding one in this case. It only works well when the CTA section is last. So in this case, it shouldn't be white, as there's not much use highlighting a resources section. We want people's attention on the preceding one. Again, something we should have in the vanilla documentation. cc @bartaz

Reply:
@lyubomir-popov then why is it white in Figma?

Reply:
shouldn't be.